<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_main</class>
 <widget class="QDialog" name="admin_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="logscreen" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>520</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="usernameText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createUser">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>create</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uti_radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>utilisateur</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin_radio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>administrateur</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dÃ©connection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
